geoposition/static/geoposition/geoposition.js

```diff
@@ -33,7 +33,7 @@ if (jQuery != undefined) {
                 $mapContainer = $('<div class="geoposition-map" />'),
                 $addressRow = $('<div class="geoposition-address" />'),
                 $searchRow = $('<div class="geoposition-search" />'),
-                $searchInput = $('<input>', {'type': 'search', 'placeholder': 'Start typing an address …'}),
+                $searchInput = $('<input>', {'type': 'text', 'placeholder': 'Start typing an address …'}),
                 $latitudeField = $container.find('input.geoposition:eq(0)'),
                 $longitudeField = $container.find('input.geoposition:eq(1)'),
                 latitude = parseFloat($latitudeField.val()) || null,
```

Comment:
For the French people who want to use this library, the value of the `$latitudeField` will have a coma so I needed to replace by a dot like so `latitude = parseFloat($latitudeField.val().replace(',', '.'))`. And do the same for the longitude.
Otherwise, the value was truncated to an integer.